gen/definitions/internal_user.yaml

```diff
@@ -45,6 +45,7 @@ attributes:
     type: Bool
     description: Whether the user is enabled/disabled
     example: true
+    mandatory: true
```

Comment:
Is there a reason you set this to "true". Based on my testing this is not required and would be set to true by default.

Reply:
I had this field in my data model, but yes you are right, its true by default. I changed it back.